iterativeTrim.py

```diff
@@ -56,8 +56,9 @@ def iterativeTrim(args,dict_dirPaths,identifier,dict_chanRegData=None,dict_calFi
             setChanRegs = True
             detName = chamber_config[ohKey]
             gemType = detName[:detName.find('-')].lower()
-            cArray_trimVal = (c_uint32*vfatsPerGemVariant[gemType]*CHANNELS_PER_VFAT)(*dict_chanRegData[ohN]["ARM_TRIM_AMPLITUDE"])
-            cArray_trimPol = (c_uint32*vfatsPerGemVariant[gemType]*CHANNELS_PER_VFAT)(*dict_chanRegData[ohN]["ARM_TRIM_POLARITY"])
+            nChannels = vfatsPerGemVariant[gemType]*CHANNELS_PER_VFAT
+            cArray_trimVal = (c_uint32*nChannels)(*dict_chanRegData[ohN]["ARM_TRIM_AMPLITUDE"])
+            cArray_trimPol = (c_uint32*nChannels)(*dict_chanRegData[ohN]["ARM_TRIM_POLARITY"])
             pass
```

Comment:
If you want to preserve the semantics of the dimensionality, you can just do, e.g.,
```python
             cArray_trimVal = (c_uint32*(vfatsPerGemVariant[gemType]*CHANNELS_PER_VFAT))(*dict_chanRegData[ohN]["ARM_TRIM_AMPLITUDE"])
```
e.g., put the argument to the `c_uint32` operation inside parentheses

Reply:
ok, done

Reply:
i'd simply squash those two commits, so there's only the one change in the history

Reply:
ok, done